### PR TITLE
SW-2735 show time zone selector in org create form

### PR DESCRIPTION
**Pull request by karthikbtf**
- required input
- use time zone when creating org
- known error: user is informed that time zone was set in info message after org switch (to be addressed separately)

Terraware App 2023-01-19 16-36-57

Terraware App 2023-01-19 16-36-44

Terraware App 2023-01-19 16-36-28

### CODE REVIEW

File: src/components/AddNewOrganizationModal.tsx [hunk 1]
```diff
@@ -159,6 +183,17 @@ export default function AddNewOrganizationModal(props: AddNewOrganizationModalPr
...
           />
         </Grid>
+        {timeZonesEnabled && (
+          <Grid item xs={12}>
+            <TimeZoneSelector
+              label={strings.TIME_ZONE_REQUIRED}
+              onTimeZoneSelected={onTimeZoneChange}
+              selectedTimeZone={newOrganization.timeZone}
+              tooltip={strings.ORGANIZATION_TIME_ZONE_TOOLTIP}
+              errorText={newOrganization.timeZone ? '' : timeZoneError}
```
Comment (charlie-tf) [inline src/components/AddNewOrganizationModal.tsx:193]: could this be
```typescript
errorText={timeZoneError}
```
?

**Review by charlie-tf — APPROVED** [review]
besides the one suggestion, looks great!

File: src/components/AddNewOrganizationModal.tsx (hunk 1 above)
Comment (karthikbtf) [reply src/components/AddNewOrganizationModal.tsx:193]: good call.. 
copy/pasta

**Comment by constanzauanini** [comment]
@karthikbtf the know issue is just the message or is it also overriding the time zone of the new organization?

**Comment by karthikbtf** [comment]
> @karthikbtf the know issue is just the message or is it also overriding the time zone of the new organization?

just the message.. Chudo is fine with that.. see https://terraformation.atlassian.net/browse/SW-2708

**Comment by constanzauanini** [comment]
> > @karthikbtf the know issue is just the message or is it also overriding the time zone of the new organization?
> 
> just the message.. Chudo is fine with that.. see https://terraformation.atlassian.net/browse/SW-2708

Sounds good, thank you!

**Pull request merged**